test-suite/run_djinni.sh

```diff
@@ -73,6 +75,35 @@ fi
     --cpp-optional-template "std::experimental::optional" \
     --cpp-optional-header "<experimental/optional>" \
     --cpp-extended-record-include-prefix "../../handwritten-src/cpp/" \
+    --cpp-use-wide-strings true \
+    \
+    --jni-out "$temp_out_relative/jni" \
+    --ident-jni-class NativeFooBar \
+    --ident-jni-file NativeFooBar \
+    \
+    --objc-out "$temp_out_relative/objc" \
```

Comment:
Why don't I see generated sources in these various output directories in this diff?  I'd expect to.
Note that we include generated output in the Djinni repo because it eases code review and makes it easier to see how a generator change affects the generated code.  It's not a best-practice for users of Djinni.